src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -1017,35 +1017,26 @@ def _parse_access_statements(nodes):
         return (default_visibility, list(explicit_public),
                 list(explicit_private))
 
-    def process_declarations(self, parent, nodes, arg_list):
+    def _process_use_stmts(self, parent, nodes):
```

Comment:
Could you make this a static method. It does not make use of self.

Reply:
Oh yes, done.